README.md

```diff
@@ -73,16 +160,53 @@ If you have a Snyk token, add that as an environment variable:
      For example, a value of `$JENKINS_HOME/tools/hudson.tasks.Maven_MavenInstallation/<Maven Name from Step 1>/bin/mvn`.
   4. Include **Invoke top-level maven targets** as a build step by specifying the Maven version, and add **clean install** as a goal for the new pipeline item.
   
-#### Option II- As a pipeline task
+#### Option 2 - As a pipeline task
 - Its just a single line that you need to add in your pipeline script.
 `rhdaAnalysis file:'manifest file path', consentTelemetry:true`
 The value description remains the same as provided in the Option I.
 User can also use the pipeline snippet generator to generate the command.
   ![](./images/pipeline.png)
+
+
+##### Example basic pipeline
+**Note: This basic declarative pipeline assumes that package managers binaries preinstalled in the pipeline' invoking machine (jenkins master or jenkins agent) **
```

Comment:
Suggested rewrite:

NOTE: The package manager binaries have to be in the pipeline's invoking machine, such as a Jenkins master or agent, for this declarative pipeline to work properly.

Reply:
Done